src/open_inwoner/js/components/card/ToggleHideSelection.js

```diff
@@ -0,0 +1,46 @@
+export class ToggleHideSelection {
+  static selector = '.card'
+
+  constructor(node) {
+    this.node = node
+    this.button = document.querySelector('#toggle-hide-elements')
+    this.button?.addEventListener('click', this.toggleHide.bind(this))
+    this.icon = document.querySelector('.readmore__icon')
+    this.node.showMoreText = document.querySelector(
+      '.showmore__button-text'
+    ).textContent
+
+    // hide all but the first 3 cards by default
+    var allCards = document.querySelectorAll('.card')
```

Comment:
While this is not an issue right now (probably), but this will select all cards on the page, which might cause issues if we reuse this somewhere else